packages/gafl-webapp-service/src/__tests__/server.spec.js

```diff
@@ -45,6 +54,34 @@ describe('The server', () => {
     await server.stop()
   })
 
+  it('addLanguageCodeToUri is called with request but not new prices', async () => {
+    createServer(catboxOptions)
+    const serverDecorateSpy = jest.spyOn(server, 'decorate').mockImplementation(jest.fn())
+
+    await init()
+    const redirect = serverDecorateSpy.mock.calls[1][2]
+    const mockRequest = { url: { pathname: '/buy' } }
+    const mockRedirect = jest.fn()
+    await redirect.call({ request: mockRequest, redirect: mockRedirect })
+
+    expect(addLanguageCodeToUri.mock.calls.some(call => call[1] === uris.NEW_PRICES.uri)).toBeFalsy()
```

Comment:
I think this might be better as `expect(addLanguageCodeToUri).not.toHaveBeenCalledWith(expect.any(Object), Uris.NEW_PRICES.uri)` (you might have to double-check the syntax on this)